doc/contribute/contributor_expectations.rst

```diff
@@ -305,10 +305,23 @@ Reviewer Expectations
   they address all non-blocking comments. PR authors should acknowledge every
   review comment in some way, even if it's just with an emoticon.
 
+- Style changes that the reviewer disagrees with but that are not documented as
```

Comment:
I would like to have in the docs what could be a clear order of precedence in terms of styling. As a maintainer of a whole platform sometimes it is necessary to apply tree wide changes that may conflict with other areas. As an example, the styling used for functions may not be the best when you need to change a register.

The intention will be clarify the precedence between SoC, driver, subsystems and libraries areas. For instance, if a vendor **maintain** the drivers possibly he wants to keep same styling for that drivers. Besides that, people have many experiences and tastes and it is a difficult topic and precedence order on this could help.

This should state too what is the precedence order about maintainer and assignee to solve the conflicts. This could avoid to have people from other areas requesting changing on styling that they should not supposed to ask. I think it is important to write in a way to not create anarchy but to help and avoid conflict resolution.

For instance, what happen when the vendor maintainer open a PR which the assignee or reviewers don't like the style?